app/controllers/RestaurantController.java

```diff
@@ -80,4 +80,19 @@ public Result makeReservation(int id){
         manager.createReservation(reservation);
         return created(JsonSerializer.serializeObject(new ReservationHelper(reservation.getReservationId())));
     }
+
+    /**
+     * Creates a Restaurant and stores it.
+     *
+     * @return the restaurantId
+     */
+    public Result createRestaurant(){
+        response().setContentType("application/json");
```

Comment:
Actually, the default response content-type is text/html, and I was looking for a way to change that, but the only way to do it was to set the content type to json in each Result of a controller. At least that is what play documentation says.